src/Microsoft.ApplicationInsights/Channel/InMemoryTransmitter.cs

```diff
@@ -138,9 +140,27 @@ private void DequeueAndSend(TimeSpan timeout)
                     // send request
                     this.Send(telemetryItems, timeout).Wait();
```

Comment:
Consider:
```csharp
this.Send(telemetryItems, timeout).GetAwaiter().GetResult();
```

Then the rest of the code can stay the same (you won't have to unpack the AggregateException)